Helper/Data.php

```diff
@@ -167,6 +184,37 @@ public function getBuyerAmount()
         return $response;
     }
 
+    /**
+     * Update Buyer Id
+     *
+     * @param $buyerId
+     */
+    public function updateBuyerId($buyerId)
```

Comment:
Let's move all buyer related functionality to more appropriate class (like `BalanceBuyer` class)